tests/conftest.py

```diff
@@ -1312,3 +1312,11 @@ def duts_minigraph_facts(duthosts, tbinfo):
         }
     """
     return duthosts.get_extended_minigraph_facts(tbinfo)
+
+@pytest.fixture(scope="module", autouse=True)
+def get_reboot_cause(duthost):
+    uptime_start = duthost.get_uptime().total_seconds()
+    yield
```

Comment:
How about using `duthost.get_up_time`? It will query the wall-clock time of reboot. We may find unexpected reboot by comparing the last reboot time and test start time.

Reply:
done